### DISCUSSION TITLE
[FEAT] Communication Hub

### DISCUSSION THREAD

**Discussion by stevew00dy** [sunflower-land/sunflower-land#4022] [Ideas]
Not everyone uses discord and people should not have to leave the game to get important updates. I propose a simple communications hub that offers the following 

![image](https://user-images.githubusercontent.com/5225886/202239025-f715a095-d2b4-43be-a185-35d5cec14573.png)

There would be three tabs available 

1, Important Updates
2, Latest News
3, Changelog

![image](https://user-images.githubusercontent.com/5225886/202239823-ed5a6a77-a955-40e9-b56c-b23c115d7f94.png)

The settings icon would have a small red dot on the top right (notifications) when there was an unread notice that would disappear only once the tab is clicked. 

This red dot would also show on the tabs to show which has unread items.

**Comment by Myth727** [comment]
Love this idea, as I have to peak around between Twitter and github. This would make my ability to stay upto date better.

**Comment by future2back** [comment]
Where is it? Better communication was promised for the new version of the game and so far nothing.


The idea is good but it should be better placed in the interface.

We also need a better UI for letterbox, should use notification and click to mark as read. It's ugly as it is.


Communication is abandoned, confused and not receiving the necessary importance. Users spend most of their time on third-party platforms/websites/sheets than the game itself.

I think that's the basics.

**Comment by AzaroQL** [comment]
![image](https://user-images.githubusercontent.com/9656961/213973535-36fd3e9f-e234-4254-8f8b-b32e21e74423.png)
![image](https://user-images.githubusercontent.com/9656961/213973558-e106f6f1-ea43-41ce-89b7-2ec916d3baf3.png)

This is used as communication hub.

**Comment by future2back** [comment]
Although the communication hub solution has been implemented, I still find it lacking. Additional tools like Discord are still necessary to gather information about the game, information that could be integrated within the game itself. In my opinion, this complicates the learning process for new players and is not user-friendly.

Can we expect improvements to the communication hub?

Thank you.
